cnames_active.js

```diff
@@ -496,6 +496,7 @@ var cnames_active = {
   "christopher": "marekkobida.github.io/christopher",
   "chromatography": "catalent.github.io/chromatography",
   "chronos": "espinielli.github.io/chronos", // noCF? (don´t add this in a new PR)
+  "ciff": "monoedev.github.io/ciff.js.org/"
```

Comment:
```suggestion
  "ciff": "monoedev.github.io/ciff.js.org",
```
Your entry needs a comma at the end of the line, and doesn't need a trailing slash :)